src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/http/MeshConfig.java

```diff
@@ -19,6 +19,7 @@
     private final String endpointPrivateKey;
     private final String subCAcert;
 
+    @SuppressWarnings("checkstyle:parameternumber")
```

Comment:
Can this be added as a filter instead of an annotation?

Reply:
Hm, not sure I agree. Methods with lots of parameters are often valid code smells; I'd rather be warned and be forced to make a judgment about if it's appropriate or needs refactoring.

Reply:
Ok, fair enough - as this annotation is only used in once place. Personally i'm not a fan of large constructors, so this is good way of forcing/reminding the author to make a judgement on this

Reply:
Yeah, I agree - I think the annotation should remain in this case. 

I did look in to refactoring the class, but decided against it because its only purpose is to fill in configurations whenever a Mesh Client is created so it makes sense sense that it has a large number of parameters in this case.